bundle/manifests/fence-agents-remediation.clusterserviceversion.yaml

```diff
@@ -117,13 +117,21 @@ spec:
         displayName: Shared Parameters
         path: template.spec.sharedparameters
       version: v1alpha1
-  description: "Fence Agents Remediation Operator (FAR)\n\nThe operator will remediate/fence
-    a node when a FenceAgentsRemediation (far) custom resource exists with the node's
-    name.\nFAR uses an existing set of upstream fencing agents for environments with
-    a \ntraditional API end-point (e.g., IPMI) for power cycling cluster nodes.\n\nThe
-    operator is recommended to be used with Node Healthcheck Operator for easier and
-    smoother experience,\nbut it can also work as a standalonde remediator for the
-    more advanced user.\n"
+  description: |
+    Fence Agents Remediation (*FAR*) is a Kubernetes operator that provides high availability for Kubernetes nodes in an automatic manner.
+    Using a management interface or traditional API, FAR runs a fence agent to remediate a node from an unhealthy state by power cycling the node.
+    By doing so it minimizes downtime for stateful applications and restores compute capacity if transient failures occur.
+
+    FAR is one of the remediator operators by [Medik8s](https://www.medik8s.io/remediation/remediation/),
+    such as [Self Node Remediation](https://github.com/medik8s/self-node-remediation) and [Machine Deletion Remediation](https://github.com/medik8s/machine-deletion-remediation),
+    that were designed to run with the Node HealthCheck Operator [(NHC)](https://github.com/medik8s/node-healthcheck-operator) as an external remediator.
```

Comment:
I think that `external remediator` in the context of NHC is confusing.
IIRC it is relevant for MHC who both have an "internal" remediation mechanism and can also use different "external" remediators.
Since NHC is not a remediator by itself it would probably be best not to use this terminology here.

Reply:
I changed it a little bit, let me know if it looks better.